cyberark/conjur-authn-k8s-client
# Print version on startup of authenticator container

This PR updates `main.go` to print the authenticator container version when it first runs. This addition accounts for different types of builds- tagged and dev builds

To test _tagged builds_, run the following:
1. Run `./bin/build`
2. With newly built image run the following: `docker run -it -i --rm conjur-authn-k8s-client:0.16.1-9e85411`
3. The following should be printed:
```
Kubernetes Authenticator Client v0.16.1-9e85411 starting up...
```

Tagged images resemble the following:
```
conjur-authn-k8s-client:0.16.1-9e85411
conjur-authn-k8s-client:0.16.1-9e85411-redhat
```

To test _dev builds_, run the following:
1. Run `env GOOS=darwin GOARCH=amd64 go build -o dist/darwin/amd64/conjur-authn-k8s-client ./cmd/authenticator`
2. Open `dist/darwin/amd64` and run executable 
3. The following should be printed:
```
INFO: 2020/03/31 07:48:53 main.go:81: Kubernetes Authenticator Client v0.16.1-dev starting up...
```

Additionally this PR removes the VERSION file, replacing it with the `pkg` version file, and updating all references.

Connected to #93
